herbert_core/classes/instrument_classes/instrument/@IX_samp/IX_samp.m

```diff
@@ -349,10 +364,12 @@
             
             S = structIndep(obj);
         end
+        %}
     end
     
     %------------------------------------------------------------------
     methods (Static)
+        %{
```

Comment:
If you make class serializable, this method should be modified according to template.
                obj = IX_samp();
                obj = loadobj@serializable(S,obj);

as there is empty constructor involved, the class can not be abstract